pandas/tests/indexes/interval/test_interval_new.py

```diff
@@ -314,3 +314,197 @@ def test_contains_method(self):
 
         assert not index.contains(20)
         assert not index.contains(-20)
+
+    def test_interval_covers_interval(self):
+
+        # class Interval:
+        #     def covers(self, other: Interval) -> bool
+
+        assert Interval(1, 3).covers(Interval(1.5, 2.5))
```

Comment:
I think `Interval` level tests would be more appropriate in `pandas/tests/scalar/test_interval.py`, which is where the other `Interval` specific tests are located.